src/util/util_env.cpp

```diff
@@ -36,7 +49,7 @@ namespace dxvk::env {
 
   std::string getExeName() {
     std::string fullPath = getExePath();
-    auto n = fullPath.find_last_of('\\');
+    auto n = fullPath.find_last_of(env::PlatformDirSlash);
```

Comment:
`std::filesystem::path::preferred_separator` ?

Reply:
This is a wchar_t on Windows, so it is not super useful here.